src/basic-syntax/static-and-const.md

```diff
@@ -1,10 +1,11 @@
 # Static and Constant Variables
 
-Global state is managed with static and constant variables.
+Static and constants are two kind of variables with special constraints.
 
 ## `const`
 
-You can declare compile-time constants:
+Constants are values created at compile time. They are inherently more efficient because they don't need to be computed at runtime
```

Comment:
Can you show me how this looks with the aspect ratio helper?

I feel you're turning a slide (with short bullet points) into a long-form text suitable for reading offline.

Reply:
Let's resolve this, we probably have worse offenders in our slides :smile: